torch_geometric/nn/conv/__init__.py

```diff
@@ -49,64 +49,20 @@
 from .hetero_conv import HeteroConv
 from .han_conv import HANConv
 from .lg_conv import LGConv
+from .disen_conv import DisenConv
 
 __all__ = [
-    'MessagePassing',
-    'GCNConv',
-    'ChebConv',
-    'SAGEConv',
-    'GraphConv',
-    'GravNetConv',
-    'GatedGraphConv',
-    'ResGatedGraphConv',
-    'GATConv',
-    'GATv2Conv',
-    'TransformerConv',
-    'AGNNConv',
-    'TAGConv',
-    'GINConv',
-    'GINEConv',
-    'ARMAConv',
-    'SGConv',
-    'APPNP',
-    'MFConv',
-    'RGCNConv',
-    'FastRGCNConv',
-    'RGATConv',
-    'SignedConv',
-    'DNAConv',
-    'PointNetConv',
-    'PointConv',
-    'GMMConv',
-    'SplineConv',
-    'NNConv',
-    'ECConv',
-    'CGConv',
-    'EdgeConv',
-    'DynamicEdgeConv',
-    'XConv',
-    'PPFConv',
-    'FeaStConv',
-    'PointTransformerConv',
-    'HypergraphConv',
-    'LEConv',
-    'PNAConv',
-    'ClusterGCNConv',
-    'GENConv',
-    'GCN2Conv',
-    'PANConv',
-    'WLConv',
-    'FiLMConv',
-    'SuperGATConv',
-    'FAConv',
-    'EGConv',
-    'PDNConv',
-    'GeneralConv',
-    'HGTConv',
-    'HEATConv',
-    'HeteroConv',
-    'HANConv',
-    'LGConv',
+    'MessagePassing', 'GCNConv', 'ChebConv', 'SAGEConv', 'GraphConv',
+    'GravNetConv', 'GatedGraphConv', 'ResGatedGraphConv', 'GATConv',
+    'GATv2Conv', 'TransformerConv', 'AGNNConv', 'TAGConv', 'GINConv',
+    'GINEConv', 'ARMAConv', 'SGConv', 'APPNP', 'MFConv', 'RGCNConv',
+    'FastRGCNConv', 'RGATConv', 'SignedConv', 'DNAConv', 'PointNetConv',
+    'PointConv', 'GMMConv', 'SplineConv', 'NNConv', 'ECConv', 'CGConv',
+    'EdgeConv', 'DynamicEdgeConv', 'XConv', 'PPFConv', 'FeaStConv',
+    'PointTransformerConv', 'HypergraphConv', 'LEConv', 'PNAConv',
+    'ClusterGCNConv', 'GENConv', 'GCN2Conv', 'PANConv', 'WLConv', 'FiLMConv',
+    'SuperGATConv', 'FAConv', 'EGConv', 'PDNConv', 'GeneralConv', 'HGTConv',
+    'HEATConv', 'HeteroConv', 'HANConv', 'LGConv', 'DisenConv'
```

Comment:
```suggestion
    'HEATConv', 'HeteroConv', 'HANConv', 'LGConv', 'DisenConv',
```

Reply:
I'm Ok with this. BTW, what benefits this comma can brings?

Reply:
It will reformat the code so that each entry is inside a new row. This is useful to easily see the diffs in code.